packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/aspnet/integrationsteps.md

```diff
@@ -424,8 +424,6 @@ switch (skipSelectionResponse.AuthenticationStatus)
 
 The method `SkipAuthenticatorSelectionAsync` can return these different response statuses: `Success` or `Terminal`. For a `Success` status, the user is signed in, the response is stored in session, and the user is redirected to the default sign-in page. In the csample app, the default sign-in page is the user profile page.
```

Comment:
Again, slightly awkward phrasing here. I'd replace "can return these different response statuses: `Success` or `Terminal`." with "can return a response status of `Success` or `Terminal`."

Also, you don't explain what the Terminal status means; only Success.

Reply:
I went back to the code to learn more about the Terminal status and it has been removed. In its place, there are now exceptions. I've updated the content to reflect the new code changes.